test/integration/endpoints/channelGateway/gateway.test.js

```diff
@@ -116,5 +141,157 @@ describe('# RiSE Channel Gateway API', () => {
           done(err)
         })
     })
+
+    it.skip('### Should Publish a Gateway', (done) => {
+
+      rise.channelGateway.list({
```

Comment:
Could we convert these from .list to the actual method?   :-)

Reply:
yep yep